src/pipecat/services/rime/tts.py

```diff
@@ -117,6 +119,8 @@ def __init__(
             else "eng",
             "speedAlpha": params.speed_alpha,
             "reduceLatency": params.reduce_latency,
+            "pauseBetweenBrackets": json.dumps(params.pause_between_brackets),
```

Comment:
I don't think json.dumps is required here as `phonemize_between_brackets` is just a bool. Same for `pause_between_brackets`.

```suggestion
            "pauseBetweenBrackets": params.pause_between_brackets,
```

Reply:
when these are stringified into url params with `"&".join(f"{k}={v}" for k, v in self._settings.items())`, python `True` becomes `True`, while the rime ws api is currently looking for `true`, which is the json bool. 

We're rolling out a change next week to be case insensitve about bool strings, but just wanted to make this patch in the mean time. 

lmk what you think. 

Given that we'll be making that change pretty soon, I'm also ok just stringifying the python bool without worrying about upper/lower case

Reply:
Got it. Makes sense. Thank you.